src/variableinspector.ts

```diff
@@ -387,27 +439,27 @@ namespace Private {
   export function createFilterTable(): HTMLDivElement {
     const container = document.createElement('div');
     container.className = 'filter-container';
-    const filterType = document.createElement('select');
+    const filterType = document.createElement('jp-select') as Select;
     filterType.className = FILTER_TYPE_CLASS;
     filterType.selectedIndex = 0;
-    const varTypeOption = document.createElement('option');
+    const varTypeOption = document.createElement('jp-option') as Option;
     varTypeOption.value = 'type';
     varTypeOption.innerHTML = 'Type';
-    const nameOption = document.createElement('option');
+    const nameOption = document.createElement('jp-option') as Option;
     nameOption.value = 'name';
     nameOption.innerHTML = 'Name';
     filterType.appendChild(varTypeOption);
     filterType.appendChild(nameOption);
     const searchContainer = document.createElement('div');
     searchContainer.className = 'jp-InputGroup filter-search-container';
-    const input = document.createElement('input');
+    const input = document.createElement('jp-search') as Search;
```

Comment:
The [jp-text-field](https://jupyterlab-contrib.github.io/jupyter-ui-toolkit/?path=/docs/components-text-field--documentation) will be more appropriate than the search component.